app/Services/UserService.php

```diff
@@ -122,4 +126,39 @@ public function loginWithGoogle($data)
         return ['user' => $user, 'token' => $token];
     }
 
+    public function requestPasswordReset($email)
+    {
+        $user = User::where('email', $email)->first();
```

Comment:
$user = User::where('email', $email)->firstOrFail(); change it to FirstOrFail() so you dont need to do the additional id(!user) check